scheduler-portal/src/test/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SchedulerServiceImplTest.java

```diff
@@ -51,6 +52,7 @@ public void setUp() throws ServletException {
         // mock servlet context
         ServletContext servletContext = mock(ServletContext.class);
         when(servletContext.getRealPath("scheduler.conf")).thenReturn(resourceAsFilePath("/scheduler.conf"));
+        when(servletContext.getRealPath("scheduler-portal.conf")).thenReturn(resourceAsFilePath("/scheduler-portal.conf"));
```

Comment:
The naming is a bit confusing. It looks like the existing `scheduler.conf` contains properties related to the Scheduler portal only. The new _PortalConfig_ class you have added and that is related to `scheduler-portal.conf` contains configuration related to Scheduler portal only. If you keep 2 different configurations, I would suggest renaming your configuration file but also the class to something more specific.